src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -1539,11 +1539,9 @@ public void onSuccess(SnapshotResponse result) {
 
 						Synapse newVersionPlace = new Synapse(entity.getId(), result.getSnapshotVersionNumber(), EntityArea.TABLES, null);
 						ToastMessageOptions.Builder messageBuilder = new ToastMessageOptions.Builder();
-						boolean dismissOnClick = true;
-						messageBuilder
-								.setTitle(SNAPSHOT_CREATED)
-								.setPrimaryButton("Go to Latest " + SNAPSHOT, () -> getGlobalApplicationState().getPlaceChanger().goTo(newVersionPlace), dismissOnClick);
+						messageBuilder.setTitle(SNAPSHOT_CREATED);
 						popupUtils.notify(SNAPSHOT_CREATED_DETAILS_TABLE, DisplayUtils.NotificationVariant.SUCCESS, messageBuilder.build());
+						getGlobalApplicationState().getPlaceChanger().goTo(newVersionPlace);
```

Comment:
Instead of adding a button to the toast message, immediately invoke PlaceChanger.goTo